assets/chat/js/chat.js

```diff
@@ -1287,14 +1287,14 @@ class Chat {
         message = new ChatMessage(messageText, null, MessageTypes.ERROR, true);
         break;
       }
-      case 'muted':
+      case 'muted': {
         this.mutedtimer.setTimer(data.muteTimeLeft);
         this.mutedtimer.startTimer();
+        const messageText = `You are temporarily muted! You can chat again ${this.mutedtimer.getReadableDuration()}. <a href="https://www.destiny.gg/subscribe">Subscribe</a>, or <a href="https://www.destiny.gg/donate">donate</a>, to remove the mute immediately.`;
```

Comment:
You shouldn't hardcode the origin in these URLs. We don't want clicking the links to navigate to destiny.gg in non-production environments. Instead, use the value in `chat.dggOrigin`.

Reply:
Links now point to dggOrigin

Reply:
Links are now relative